pkg/cloud/powervs/assets/deployment.yaml

```
@@ -77,7 +73,7 @@ spec:
               source /etc/kubernetes/apiserver-url.env
             fi
             exec /bin/ibm-cloud-controller-manager \
-            --bind-address=$(POD_IP_ADDRESS) \
+            --bind-address=$(127.0.0.1) \
```

This is a literal not a variable
```suggestion
            --bind-address=127.0.0.1 \
```

---

stupid mistake!